back-end/src/controllers/MainController.ts

```diff
@@ -69,10 +75,34 @@ export const confirm = async (req: express.Request, res: express.Response) => {
 	}
 }
 
+export const submitToken = async (req: express.Request, res: express.Response) => {
+	try{
+		let pubkey = fs.readFileSync('certificates/backend_cert.pem');
+		let validToken = jwt.verify(req.body.token,pubkey); // check token's expiry
+		if(validToken){
+			//creating new verification token entry in the database
+			const newToken = new VerificationTokenModel({
+				value: req.body.token
+			});
+
+			// save the new verification token
+			await newToken.save();
```

Comment:
A couple of things here:

1. Should save a hash of the token to the DB, not the token itself. 
2. Before saving, should check to make sure the hash doesn't already exist in the DB.

Reply:
I can still use this code in my video because I won't be showing exactly what newToken.save() does.